hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeysPublic.java

```diff
@@ -494,6 +494,10 @@
                                                 "ipc.server.log.slow.rpc";
   public static final boolean IPC_SERVER_LOG_SLOW_RPC_DEFAULT = false;
 
+  public static final String IPC_SERVER_PURGE_INTERVAL_NANOS_MINUTES_KEY =
```

Comment:
sorry missed this one:
we should rename this variable. It is in minutes and nothing to do with nanoseconds.
Call it IPC_SERVER_PURGE_INTERVAL_MINUTES_KEY? same for IPC_SERVER_PURGE_INTERVAL_NANOS_DEFAULT